example/docker-helper/docker_helper/utils.py

```diff
@@ -35,18 +36,39 @@ def output_lines(raw_output, encoding='utf-8'):
     return raw_output.decode(encoding).splitlines()
 
 
-def list_container_processes(container, columns=['pid', 'ruser', 'args']):
-    # We use an exec here rather than `container.top()` because we want to run
-    # 'ps' inside the container. This is because we want to get PIDs and
-    # usernames in the container's namespaces. `container.top()` uses 'ps' from
-    # outside the container in the host's namespaces. Note that this requires
-    # the container to have a 'ps' that responds to the arguments we give it.
-    cols = ','.join(columns)
-    ps_output = container.exec_run(['ps', 'ax', '-o', cols])
-    ps_lines = output_lines(ps_output)
-    ps_lines.pop(0)  # Skip the header
-    ps_entries = [line.split(None, max(0, len(columns) - 1))
-                  for line in ps_lines]
-    # Drop the entry for the ps command itself
-    ps_entries = [e for e in ps_entries if e[2] != 'ps ax -o {}'.format(cols)]
-    return ps_entries
+PsRow = namedtuple('PsRow', ['pid', 'ruser', 'args'])
```

Comment:
These are just the values we use already. The idea is to add more in future PRs.